safe/impact_functions/earthquake/itb_earthquake_fatality_model/impact_function.py

```diff
@@ -115,7 +115,7 @@ def __init__(self):
                 1: 0, 2: 0, 3: 0, 4: 0, 5: 0, 6: 1.0,
                 7: 1.0, 8: 1.0, 9: 1.0, 10: 1.0
             }),
-            ('mmi_range', range(2, 10)),
+            ('mmi_range', range(4, 11)), # from MMI 4 to 10
```

Comment:
yes, but we also need to calculate exposed people from the range [2,10]. Are you suggesting that we should also show the exposed people from the MMI 4 in the report?